.github/workflows/frontend-workflow.yml

```diff
@@ -133,6 +133,36 @@ jobs:
           npm ci --loglevel verbose
         fi
 
+    - name: Validate specified deps not duplicated
+	if: ${{ inputs.custom_deps_not_dup == 'true' }}
+	run: |
+            NPM=${{ inputs.pm }}
+
+	    if test "$NPM" = "npm"; then
```

Comment:
alias `may_tail` is a working solution but I would prefer to have 2 sets of functions for better readibility:
`dupe_check_npm()`
`dupe_check_other()`

each one being called in a dedicated task of the workflow maybe:
`- name: Validate specified deps not duplicated with NPM`
`- name: Validate specified deps not duplicated with other`